app/controllers/reviews_controller.rb

```diff
@@ -18,6 +20,42 @@ def create
     end
   end
 
+  def upvote
+    @review = Review.find(params[:id])
+    @review.upvote_by current_user
+
+    respond_to do |format|
+      if @review.upvote_by current_user
+        format.json {
+          render json: {
+          id: @review.id,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.